### PR TITLE
Add header explanation for Laravel Nova

### PR DESCRIPTION
**Pull request by CodeTechNL**
I've been searching why my Laravel Nova wasn't working. Some extra explanation for the docs should help others out

### CODE REVIEW

**Comment by afu-dev** [comment]
In the file you modified, we already have a list of "other headers": https://github.com/brefphp/bref/blob/master/docs/frameworks/laravel.md?plain=1#L181-L182

Shouldn't we add the `X-Inertia` in the same list?

**Comment by CodeTechNL** [comment]
@afu-dev  I’ve added it as optional. Not everyone uses Nova or Inertia. There is a limit of 9 (if I’m right) on the forwarded headers.

**Comment by afu-dev** [comment]
Likewise, not everyone uses Livewire. I propose to have all optional headers in the same place instead of one in the `forwardedHeaders` list and one underneath the source code.

Indeed, the `forwardedHeaders` list is limited (to 10 headers, ref: https://github.com/getlift/lift/blob/master/docs/server-side-website.md?plain=1#L365), and adding everything to it will break the code. Maybe a note like you did with "if you use this, add this header, if you use that, add that header"?

(Again, just a proposition to be extra clear with everyone reading the doc 😉)

**Comment by CodeTechNL** [comment]
I agree on it @afu-dev, I’ll update it later today and change livewire too. Also I’ll add a notice about the limit of 10 forwardedHeaders

**Comment by mnapoli** [comment]
Thanks for the PR, and good points here!

How about adding it to the list, but commented (with a comment above saying "uncomment if you use Nova"). We could even do the same thing for the Livewire header (commented by default), that way we avoid hitting the limit of 10 headers too easily?

**Comment by CodeTechNL** [comment]
@mnapoli ill see what’s the best to keep it clear and brief and update my PR

**Comment by CodeTechNL** [comment]
@mnapoli little bit late, kinda busy. Updated as requested

File: docs/frameworks/laravel.md [hunk 1]
```diff
@@ -178,10 +178,15 @@ constructs:
       - X-Requested-With
       # Laravel Framework Headers
       - X-Csrf-Token
-      # Other Headers, e.g. Livewire
-      - X-Livewire
+      # Other Headers, e.g. Livewire, Laravel Nova
```
Comment (afu-dev) [inline docs/frameworks/laravel.md:181]:
```suggestion
      # Other Headers (e.g. Livewire, Laravel Nova), uncomment based on your needs
```

File: docs/frameworks/laravel.md [hunk 2]
```diff
@@ -178,10 +178,15 @@ constructs:
...
+      # - X-Livewire
+      # - X-Inertia
 ```
 
+> If you use Laravel Nova, don't forget to uncomment the header `X-Inertia`
+
```
Comment (afu-dev) [inline docs/frameworks/laravel.md:187]:
```suggestion
```

**Review by afu-dev — CHANGES_REQUESTED** [review]
The note is specific for Laravel Nova, but the code now contains headers for Livewire AND Nova. I would make the comment in the YAML more detailed and remove the note.

**Comment by mnapoli** [comment]
Thank you both for contributing!

**Pull request merged**